### ISSUE TITLE
Build fails with Cabal-3.0

### ISSUE THREAD

**Issue by abhin4v** [jakubfijalkowski/hlibsass#12]
Build fails with Cabal-3.0 with this error:

```
/private/var/folders/3b/9plp3gb959gcv_nyqg9j_k740000gn/T/stack45489/hlibsass-0.1.8.0/Setup.hs:41:13: error:
     Not in scope: ‘preSDist’
    |
 41 |           , preSDist = updateLibsassVersion
    |             ^^^^^^^^
```

This seems to be because of using the [now removed](https://github.com/haskell/cabal/commit/3d9a49073d71b2597662a389406a94fcb9b399b2#diff-4c224b398e56634dcc9ee65f15caee8eL110) `preSDist` hook.

**Comment by jakubfijalkowski** [comment]
Thanks for reporting!

I will try to look into it this week (-end is more probable).

**Comment by jakubfijalkowski** [comment]
Should be fixed with the recently released v0.1.8.1.

**Comment by abhin4v** [comment]
Thanks for quick fix @jakubfijalkowski
